pages/governance.md

```diff
@@ -0,0 +1,179 @@
+[Bordeaux, 2018](https://jump.dev/meetings/bordeaux2018/), and [Boston, 2017](https://jump.dev/meetings/mit2017/).
+
+Announcements of future workshops will be communicated through the
+[Community forum] and via the [jump.dev](https://jump.dev) website.
+
+## Decision making process
+
+### Technical
+
+Technical decisions, such when a pull request is ready to be accepted and
+merged, are made by consensus via the relevant Github issues and pull requests.
+If consensus cannot be achieved, the community member proposing the change may
+be invited by the Steering Council to present their proposal at a Monthly
+developer call for further discussion and community input.
+
+Due to his role in the creation of JuMP, Miles Lubin holds the role of [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).
+If consensus cannot be achieved, the final decision shall be made by the BDFL.
+
+### Other
+
+All other decisions to be made, e.g., the date and location for a JuMP-dev
+workshop, should be tabled as an agenda item and discussed on a Monthly
+developer call. If consensus cannot be achieved, the final decision shall be
+made by the Steering Council.
+
+## Commit rights
+
+Commit rights are managed through [Github permissions](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization)
+in the [JuMP-dev Github organization].
+
+There are two levels of commit rights: repository maintainers and core
+contributors.
+
+For security, all maintainers and core contributors must have two-factor
+authentication enabled through their Github account.
+
+### Repository Maintainer
+
+Repository maintainers have `Maintain` rights to an individual repository as an
+outside collaborator. This allows them to push code to a branch in the
+`jump-dev` organization instead of their personal fork, merge pull requests, and
+close issues.
+
+Community members can become a repository maintainer by actively engaging in the
+development and maintenance of a repository for a sustained period of time. This
+includes triaging issues, proposing and reviewing pull requests, and updating
+any binary dependencies as needed. Once the member judges that they are an
+integral part of the repository, they can apply to the Steering Council to
+become a repository maintainer. The Steering Council will make a determination
+based on the length of time that the member has been involved with the
+repository, and the depth of their contributions. The Steering Council may also,
+at any time, offer a community member the choice to become a maintainer.
+
+Community members can be maintainers of multiple repositories at the same time.
+
+### Core Contributor
+
+Core contributors are added as members of the [JuMP-dev Github organization] and
+have `Admin` permission to every repository.
+
+Community members can become core contributors by invite from the Steering
```

Comment:
I would say that a new core contributor should be added by consensus of the current core contributors and notification to the steering council. I don't think the steering council needs to get involved in technical decisions unless they're strategically significant.